packages/utils/src/enumOptionsDeselectValue.ts

```diff
@@ -1,4 +1,4 @@
-import isEqual from 'lodash/isEqual';
+import { deepEqual } from 'fast-equals';
```

Comment:
Can you switch these to all use the `@rjsf/utils` `deepEquals()` first? And then in a second PR switch `deepEquals` to use this function?